spec/client_spec.rb

```diff
@@ -101,13 +101,18 @@
       end
 
       it 'should include the engine name in listed objects' do
-        # Create an engine
         client.create_engine(engine_name)
 
-        # Get the list
         engines = client.list_engines
         expect(engines.find { |e| e['name'] == engine_name }).to_not be_nil
       end
+
+      it 'should include the engine name in listed objects with pagination' do
+        client.create_engine(engine_name)
+
+        engines = client.list_engines(:page => 1, :per_page => 20)
+        expect(engines.find { |e| e['name'] == engine_name }).to_not be_nil
```

Comment:
this isn't actually the expected response - the expected response includes `meta` and `results` as top level keys